src/App.js

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 
+const todoList = [{id:1,title:"complete assignment",},{id:2,title:"pick up kid"},{id:3,title:"sleep"}]
```

Comment:
let's clean this up a bit

```
const todoList = [
  { id:1, title: "complete assignment" },
  { id:2, title: "pick up kid"},
  { id:3,title:"sleep" }
]
```

- Do not add a trailing comma if your object is a one-liner
- Create spaces to make your code more readable. Remember that once you start collaborating with others readability of your code is important. Let's try practicing from the beginning of this course.

Definitely recommend installing a code formatter like prettier and running it every time you save (you can set your IDE to do this automatically).